lms/templates/instructor/instructor_dashboard_2/student_admin.html

```diff
@@ -85,6 +85,49 @@
   <hr>
 </div>
 
+% if course.entrance_exam_enabled:
+<div class="entrance-exam-grade-container action-type-container">
+  <h2>${_("Entrance Exam grade adjustment")}</h2>
+  <div class="request-response-error"></div>
+  <p>
```

Comment:
@andy-armstrong @ziafazal, since the help text is functionally equivalent to the ```<h2>``` and is so conveniently contained in a ```<p>``` that happens to also contain the form input, I would convert all of the ```<p>``` elements (that wrap and describe the form inputs) to ```<label>``` elements.  Otherwise, we will have to add unique ID's to all of those headers and aria-labelledby attributes to all of the form inputs.